### ISSUE TITLE
Can't paste HTML tags

### ISSUE THREAD

**Issue by dharik** [TeamCodeStream/CodeStream#158]
**Describe the bug**
Pasted text  goes through a strip tags process so text surrounded by `<` and `>` isn't paste-able. Pasting `<HEllo> sdfsdf` strips out the `<HEllo>`

**System Info (please complete the following information):**
 - Version of the CodeStream extension: [e.g., 0.53.1] 6.2.0
 - OS + version: [e.g. Win 10] MacOS Catalina 10.15.3
 - IDE + version: [e.g. VS Code 1.33.1] VS Code Version: 1.41.1

**Log Files**

Output: codestream
```
[2020-03-05 13:56:11:965] Webview: Received message codestream/codemark/sha1(wv:60:Kzd_fhZjT7CP7u-aGH3h9w) from the webview
[2020-03-05 13:56:11:965] AgentConnection.sendRequest(codestream/codemark/sha1) — {"codemarkId":"5e6101abab21d135b1ac57bc","markerId":"5e6101abab21d135b1ac57bd"}
[2020-03-05 13:56:11:966] WebviewPanel: Completed posting response(wv:60:Kzd_fhZjT7CP7u-aGH3h9w) to the webview
[2020-03-05 13:56:11:968] Webview: Received message codestream/textDocument/fromMarker(wv:61:vfuUx3yQQL2JwTALlbxn4w) from the webview
[2020-03-05 13:56:11:968] AgentConnection.sendRequest(codestream/textDocument/fromMarker) — {"markerId":"5e6101abab21d135b1ac57bd"}
[2020-03-05 13:56:11:969] WebviewPanel: Completed posting response(wv:61:vfuUx3yQQL2JwTALlbxn4w) to the webview
[2020-03-05 13:56:12:967] Webview: Received message codestream/codemark/sha1(wv:62:95tMZmwzQ2GG1KJ4mkl7cA) from the webview
[2020-03-05 13:56:12:967] AgentConnection.sendRequest(codestream/codemark/sha1) — {"codemarkId":"5e6101abab21d135b1ac57bc","markerId":"5e6101abab21d135b1ac57bd"}
[2020-03-05 13:56:12:968] WebviewPanel: Completed posting response(wv:62:95tMZmwzQ2GG1KJ4mkl7cA) to the webview
[2020-03-05 13:56:12:970] Webview: Received message codestream/textDocument/fromMarker(wv:63:AzQw8pDESleikIcPdnoZ8g) from the webview
[2020-03-05 13:56:12:970] AgentConnection.sendRequest(codestream/textDocument/fromMarker) — {"markerId":"5e6101abab21d135b1ac57bd"}
[2020-03-05 13:56:12:971] WebviewPanel: Completed posting response(wv:63:AzQw8pDESleikIcPdnoZ8g) to the webview
[2020-03-05 13:56:13:928] WebviewPanel: Completed posting webview/editor/didChangeSelection to the webview
[2020-03-05 13:56:13:964] Webview: Received message codestream/codemark/sha1(wv:64:KTd4fmR8RdWit_vP459Leg) from the webview
[2020-03-05 13:56:13:965] AgentConnection.sendRequest(codestream/codemark/sha1) — {"codemarkId":"5e6101abab21d135b1ac57bc","markerId":"5e6101abab21d135b1ac57bd"}
[2020-03-05 13:56:13:967] WebviewPanel: Completed posting response(wv:64:KTd4fmR8RdWit_vP459Leg) to the webview
[2020-03-05 13:56:13:970] Webview: Received message codestream/textDocument/fromMarker(wv:65:Q6hifDIpQnmajTcs5u68bw) from the webview
[2020-03-05 13:56:13:970] AgentConnection.sendRequest(codestream/textDocument/fromMarker) — {"markerId":"5e6101abab21d135b1ac57bd"}
[2020-03-05 13:56:13:972] WebviewPanel: Completed posting response(wv:65:Q6hifDIpQnmajTcs5u68bw) to the webview
[2020-03-05 13:56:14:179] WebviewPanel: Completed posting webview/editor/didChangeSelection to the webview
[2020-03-05 13:56:14:484] WebviewPanel: Completed posting webview/editor/didChangeSelection to the webview
[2020-03-05 13:56:14:965] Webview: Received message codestream/codemark/sha1(wv:66:UGQib842S56q6uW_AcxJ3Q) from the webview
[2020-03-05 13:56:14:966] AgentConnection.sendRequest(codestream/codemark/sha1) — {"codemarkId":"5e6101abab21d135b1ac57bc","markerId":"5e6101abab21d135b1ac57bd"}
[2020-03-05 13:56:14:968] WebviewPanel: Completed posting response(wv:66:UGQib842S56q6uW_AcxJ3Q) to the webview
[2020-03-05 13:56:14:971] Webview: Received message codestream/textDocument/fromMarker(wv:67:1AsoaEkZTIKNEjeKlYhxqQ) from the webview
[2020-03-05 13:56:14:971] AgentConnection.sendRequest(codestream/textDocument/fromMarker) — {"markerId":"5e6101abab21d135b1ac57bd"}
[2020-03-05 13:56:14:972] WebviewPanel: Completed posting response(wv:67:1AsoaEkZTIKNEjeKlYhxqQ) to the webview
[2020-03-05 13:56:15:965] Webview: Received message codestream/codemark/sha1(wv:68:l1jYzBZpRZOjpuX8YBPsrg) from the webview
[2020-03-05 13:56:15:965] AgentConnection.sendRequest(codestream/codemark/sha1) — {"codemarkId":"5e6101abab21d135b1ac57bc","markerId":"5e6101abab21d135b1ac57bd"}
[2020-03-05 13:56:15:967] WebviewPanel: Completed posting response(wv:68:l1jYzBZpRZOjpuX8YBPsrg) to the webview
[2020-03-05 13:56:15:970] Webview: Received message codestream/textDocument/fromMarker(wv:69:5ZP2JNZqTjmDQC__5JpwGQ) from the webview
[2020-03-05 13:56:15:970] AgentConnection.sendRequest(codestream/textDocument/fromMarker) — {"markerId":"5e6101abab21d135b1ac57bd"}
[2020-03-05 13:56:15:971] WebviewPanel: Completed posting response(wv:69:5ZP2JNZqTjmDQC__5JpwGQ) to the webview
[2020-03-05 13:56:16:967] Webview: Received message codestream/codemark/sha1(wv:70:r92jeKp4RYKIY-r69srSJw) from the webview
[2020-03-05 13:56:16:967] AgentConnection.sendRequest(codestream/codemark/sha1) — {"codemarkId":"5e6101abab21d135b1ac57bc","markerId":"5e6101abab21d135b1ac57bd"}
[2020-03-05 13:56:16:969] WebviewPanel: Completed posting response(wv:70:r92jeKp4RYKIY-r69srSJw) to the webview
[2020-03-05 13:56:16:971] Webview: Received message codestream/textDocument/fromMarker(wv:71:tCLooXkzTx6D5bYg_aHVJg) from the webview
[2020-03-05 13:56:16:971] AgentConnection.sendRequest(codestream/textDocument/fromMarker) — {"markerId":"5e6101abab21d135b1ac57bd"}
[2020-03-05 13:56:16:973] WebviewPanel: Completed posting response(wv:71:tCLooXkzTx6D5bYg_aHVJg) to the webview
[2020-03-05 13:56:17:968] Webview: Received message codestream/codemark/sha1(wv:72:BeOfRCOAT8WSxMwTgVQpcg) from the webview
[2020-03-05 13:56:17:968] AgentConnection.sendRequest(codestream/codemark/sha1) — {"codemarkId":"5e6101abab21d135b1ac57bc","markerId":"5e6101abab21d135b1ac57bd"}
[2020-03-05 13:56:17:970] WebviewPanel: Completed posting response(wv:72:BeOfRCOAT8WSxMwTgVQpcg) to the webview
[2020-03-05 13:56:17:972] Webview: Received message codestream/textDocument/fromMarker(wv:73:MENC5jwzToOQeHJxmGicPA) from the webview
[2020-03-05 13:56:17:972] AgentConnection.sendRequest(codestream/textDocument/fromMarker) — {"markerId":"5e6101abab21d135b1ac57bd"}
[2020-03-05 13:56:17:973] WebviewPanel: Completed posting response(wv:73:MENC5jwzToOQeHJxmGicPA) to the webview
[2020-03-05 13:56:18:966] Webview: Received message codestream/codemark/sha1(wv:74:6MeoWo5oTjiV_P59MxUPIQ) from the webview
```

Output: codestream (agent)
```
[2020-03-05 13:57:46:203] [2b3] CodemarksManager.getCodemarkSha1 — {"codemarkId":"5e6101abab21d135b1ac57bc","markerId":"5e6101abab21d135b1ac57bd"}, {"_isCancelled":false}
[2020-03-05 13:57:46:204] [692] CodemarkEntityCache.get completed • 0 ms — e=[["id","5e6101abab21d135b1ac57bc"]], t=undefined
[2020-03-05 13:57:46:204] [693] MarkerEntityCache.get completed • 0 ms — e=[["id","5e6101abab21d135b1ac57bd"]], t=undefined
[2020-03-05 13:57:46:204] [694] FileStreamEntityCache.get completed • 0 ms — e=[["id","5e6101abab21d135b1ac57be"]], t=undefined
[2020-03-05 13:57:46:204] [2b3] CodemarksManager.getCodemarkSha1 completed • 0 ms
[2020-03-05 13:57:46:220] [695] MarkerEntityCache.get completed • 0 ms — e=[["id","5e6101abab21d135b1ac57bd"]], t=undefined
[2020-03-05 13:57:47:006] [2b8] CodemarksManager.getCodemarkSha1 — {"codemarkId":"5e6101abab21d135b1ac57bc","markerId":"5e6101abab21d135b1ac57bd"}, {"_isCancelled":false}
[2020-03-05 13:57:47:006] [697] CodemarkEntityCache.get completed • 0 ms — e=[["id","5e6101abab21d135b1ac57bc"]], t=undefined
[2020-03-05 13:57:47:006] [698] MarkerEntityCache.get completed • 0 ms — e=[["id","5e6101abab21d135b1ac57bd"]], t=undefined
[2020-03-05 13:57:47:006] [699] FileStreamEntityCache.get completed • 0 ms — e=[["id","5e6101abab21d135b1ac57be"]], t=undefined
[2020-03-05 13:57:47:006] [2b8] CodemarksManager.getCodemarkSha1 completed • 0 ms
[2020-03-05 13:57:47:009] [700] MarkerEntityCache.get completed • 0 ms — e=[["id","5e6101abab21d135b1ac57bd"]], t=undefined
[2020-03-05 13:57:47:967] [2bd] CodemarksManager.getCodemarkSha1 — {"codemarkId":"5e6101abab21d135b1ac57bc","markerId":"5e6101abab21d135b1ac57bd"}, {"_isCancelled":false}
[2020-03-05 13:57:47:967] [702] CodemarkEntityCache.get completed • 0 ms — e=[["id","5e6101abab21d135b1ac57bc"]], t=undefined
[2020-03-05 13:57:47:967] [703] MarkerEntityCache.get completed • 0 ms — e=[["id","5e6101abab21d135b1ac57bd"]], t=undefined
[2020-03-05 13:57:47:967] [704] FileStreamEntityCache.get completed • 0 ms — e=[["id","5e6101abab21d135b1ac57be"]], t=undefined
[2020-03-05 13:57:47:968] [2bd] CodemarksManager.getCodemarkSha1 completed • 0 ms
[2020-03-05 13:57:47:973] [705] MarkerEntityCache.get completed • 0 ms — e=[["id","5e6101abab21d135b1ac57bd"]], t=undefined
[2020-03-05 13:57:48:970] [2c2] CodemarksManager.getCodemarkSha1 — {"codemarkId":"5e6101abab21d135b1ac57bc","markerId":"5e6101abab21d135b1ac57bd"}, {"_isCancelled":false}
[2020-03-05 13:57:48:970] [707] CodemarkEntityCache.get completed • 0 ms — e=[["id","5e6101abab21d135b1ac57bc"]], t=undefined
[2020-03-05 13:57:48:970] [708] MarkerEntityCache.get completed • 0 ms — e=[["id","5e6101abab21d135b1ac57bd"]], t=undefined
[2020-03-05 13:57:48:970] [709] FileStreamEntityCache.get completed • 0 ms — e=[["id","5e6101abab21d135b1ac57be"]], t=undefined
[2020-03-05 13:57:48:970] [2c2] CodemarksManager.getCodemarkSha1 completed • 0 ms
[2020-03-05 13:57:48:975] [710] MarkerEntityCache.get completed • 0 ms — e=[["id","5e6101abab21d135b1ac57bd"]], t=undefined
[2020-03-05 13:57:49:966] [2c7] CodemarksManager.getCodemarkSha1 — {"codemarkId":"5e6101abab21d135b1ac57bc","markerId":"5e6101abab21d135b1ac57bd"}, {"_isCancelled":false}
```

**Comment by planteater** [comment]
@dharik - When you include html, or code of any kind, in a message you'll need to format it as code using triple backticks on either side (i.e., just as you would in Slack). Can you give that a sot and let me know if it does the trick?

**Comment by planteater** [comment]
@dharik - Ignore my previous reply. I misunderstood the issue initially.

**Comment by jpringle11** [comment]
I have encountered similar issues .. both with comments in code backticks as well as "bare" comments.

Simplest example - I want to describe a parameterized type `List<String>`.

If it's inline, the first time I put it either bare (no formatting) or within single back ticks, it will show correctly.  However, if I then edit the comment, the <String> part is removed.

When using triple backticks, it is stripped in the formatted view

> ```
> List<String>
> ```
> 

shows simply as
```
List
```

and editing the comment shows the `<String>` has been removed.

**Comment by jpringle11** [comment]
It gets even more funky with wildcards for type bindings - I couldn't figure out a way to get `List<?>` to render correctly.

**Comment by ppezaris** [comment]
Thanks for the reports -- we should have a fix for these issues in our next release.

**Comment by planteater** [comment]
@dharik @jpringle11 - These issues should be resolved in v7.2.2 of the CodeStream extension, which was just released.
